### ISSUE TITLE
Support PHP language

### ISSUE THREAD

**Issue by ricpelo** [hplgit/doconce#142]
DocOnce supports several programming languages (C, Python, Ruby...) but lacks support for PHP language.

Is there an easy way to include PHP language support?

Thanks in advance.

**Comment by KGHustad** [comment]
It is supported. See the [list of available lexers](http://pygments.org/languages/).

```
!bc php
<?php
  echo "Hello World!";
?>
!ec
```

**Comment by ricpelo** [comment]
When I try the above using `doconce format pdflatex hello.do.txt --xelatex --latex_code_style=pyg --latex_admon=grayicon`, I get:

```latex
\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
<?php
echo "Hello World!";
?>                                                    
\end{minted}                                                                    
```

I get no syntax highlighting.

I'm using DocOnce version 1.4.4.

**Comment by KGHustad** [comment]
Interesting. Apparently, there's some special handling for LaTeX. Will look into it.

**Comment by KGHustad** [comment]
Should be fixed with d304ea2f
